### DISCUSSION TITLE
Escort distance problem

### DISCUSSION THREAD

**Discussion by w5860363** [TrinityCore/TrinityCore#27755] [Q&A] [answered]
http://www.wowhead.com/quest=12801 
Affected by the escort distance
If you stay away from Darion Mograine, the quest will be interrupted and fail without prompt The show will not go on
And as far as I know, in Blizzard's server
Even standing at the starting point of quest can participate in it
----------------------
How to modify and exclude the quest from the escort system?

**Comment by offl** [comment, marked as answer]
SetDespawnAtFar(false);

**Reply by w5860363** [reply]
Thanks
